packages/node/src/builders/execute/schema.json

```diff
@@ -25,6 +25,11 @@
       "default": 7777,
       "description": "The port to inspect the process on"
     },
+    "restart": {
```

Comment:
Please rename this option to `watch` to match `@nrwl/web:dev-server` and `@angular-devkit/build-angular:dev-server`.

Reply:
Are you sure? This PR is about the `@nrwl/node`.

Reply:
Yes, I think the consistency is important 😉.